models/User.js

```diff
@@ -156,6 +156,8 @@ User.beforeValidate(async (user) => {
     if (typeof user.phone === 'string') user.phone = user.phone.trim();
     if (typeof user.address === 'string') user.address = user.address.trim();
     if (typeof user.about_me === 'string') user.about_me = user.about_me.trim();
+
```

Comment:
This line can be removed imo

Reply:
yeah, I've left it there to split plain strings processing from date processing, it's not exactly the same